pkg/runner/runner.go

```diff
@@ -245,8 +257,17 @@ func Run(cfg *Config) error {
 				stopRuler()
 			},
 		)
+	} else {
+		log.Info("msg", "Rules files not found in the given Prometheus configuration file. Both rule-manager and alerting will not be initialized")
 	}
 
+	// Asynchronously update the telemetry information.
+	go func() {
```

Comment:
How will this work with hot-reloading of rules? Or maybe you plan to refactor later on?

Reply:
Right now, we are not thinking of telemetry with hot-reloading. Hot-reloading will be an architectural change, hence thinking right now is not good.